index.js

```diff
@@ -369,6 +369,18 @@ function Jasmine2HTMLReporter(options) {
         exportObject.endTime = new Date();
     };
 
+    self.afterLaunch = function(callback) {
+        // write success report. if all test pass and showFailuresOnly options is true,
+        // file wont be created.
```

Comment:
I'm confused, it appears that on line 377, if showFailuresOnly is true, the file WILL be created.

Reply:
Yeah... its supposed to mean that currently the file is not created if both options are true. that is a confusing comment. Let me update that. Thanks.